source/Octopus.Cli/Commands/CompleteCommand.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Octopus.Cli.Infrastructure;
+using Octopus.Cli.Model;
+using Octopus.Cli.Util;
+
+namespace Octopus.Cli.Commands
+{
+    [Command("complete", Description = "Find the most likely subcommand completion based on a partial subcommand.")]
```

Comment:
This will need change once you've added the `--install` option, right?

Reply:
:thinking: Not sure what you mean. It still operates the same way, post install command impl (coming very soon). The scripts that get installed just make use of it.

Reply:
I'm thinking that i would the usage of the `complete` command to be:
`octo complete he` => returns commands starting with `he`
`octo complete --install zsh` => installs the shell completion hook for zsh

if I'm gazing into the same 🔮 as you... then this would need to say 

```
    [Command("complete", Description = "Supports command line auto completion.")]
```
to be generic enough to cover both descriptions?

Reply:
ah ok, I went with a different command to install completions them. 
For e.g. `octo install-autocomplete --shell=bash`

Reply:
The two concerns felt very seperate: one is all about filesystems and userprofiles, the other is surfacing suggestions